service/matching/tasklist/task_list_manager.go

```diff
@@ -253,6 +267,46 @@ func (c *taskListManagerImpl) handleErr(err error) error {
 	return err
 }
 
+func (c *taskListManagerImpl) reloadTaskListPartitionConfig() {
```

Comment:
`reloadTaskListPartitionConfig` is not actually reloading. it is initializing if not set